modules/custom/express_help/express_help.module

```diff
@@ -112,9 +112,13 @@ function _express_help_list() {
       ),
     ),
   );
+  $module_owner = module_implements('express_help');
+    foreach ($module_owner as $owner) {
+      t($owner);
```

Comment:
I believe the t() function returns a string you must set as a variable. Somehow, all I saw was "cu_social_media_bundle" for that value.